libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1315,8 +1315,8 @@ AP_GPS_UBLOX::_parse_gps(void)
         break;
     case MSG_RELPOSNED:
         {
-            const Vector3f &offset0 = gps._antenna_offset[0].get();
-            const Vector3f &offset1 = gps._antenna_offset[1].get();
+            const Vector3f &offset0 = gps._antenna_offset[state.instance^1].get();
```

Comment:
I feel like this is a future bug for when we support a third GPS. How about:
```
((state.instance + 1) % GPS_MAX_RECEIVERS)
((state.instance) % GPS_MAX_RECEIVERS)
```

Reply:
that would just give incorrect data. We'd need a method to pair the rover with the base.